pkg/codegen/testing/test/sdk_driver.go

```diff
@@ -451,6 +451,11 @@ var PulumiPulumiSDKTests = []*SDKTest{
 		Description: "Testing the kubernetes20 compatibility mode.",
 		Skip:        codegen.NewStringSet("go/test"),
 	},
+	{
+		Directory:   "regress-dotnet-14130",
+		Description: "Test for https://github.com/pulumi/pulumi/issues/14130",
+		Skip:        allLanguages.Except("dotnet/any"),
```

Comment:
Why are we not running this test for all languages? I'm hesitant to accept and say we support props like "$foo" if we aren't testing that everywhere.

Reply:
> Also I inherited this from the previous PR that I picked up to finish it off.

And what I mean by that is, I didn't intentionally scope this test to run only against dotnet but that that's how it was but it kind of makes sense because the issue I created is specific to output types in C#. I don't know if the other languages have this problem too. If they do, it'll have to be a separate issue.